client/my-sites/marketing/tools/feature.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode, FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+
+interface MarketingToolsFeatureProps {
+	children: ReactNode;
+	description: string;
+	disclaimer?: string;
+	imageAlt?: string;
+	imagePath?: string;
+	title: string;
+}
```

Comment:
in https://github.com/Automattic/wp-calypso/pull/32689/files#r279961884 I made a comment about the name of this. in short: do we gain much by choosing this long name whose context is already known by virtue of the fact that it's in the `marketing/tools/feature` module rather than picking something inherently simpler like `Props`?

Reply:
I think the verbose names were parroted from whatever example I used for Typescript with React. The shorter names are much more readable to me!